docs/source/rbac.rst

```diff
@@ -1,14 +1,6 @@
 Role Based Access Control
 =========================
 
-.. note::
-
-  Prior to StackStorm 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform which included Role Based Access Control (RBAC).
-
-Extreme Workflow Composer was donated to Linux Foundation, but those components are not yet integrated into the StackStorm release. Please visit https://stackstorm.com/2020/05/27/extreme-networks-donates-ewc-to-linux-foundation/ for more details.
-
-  These components are NOT available in the StackStorm 3.3 release, but will be integrated into the community edition of StackStorm in future release(s).
-
 Role Based Access Control (RBAC) allows system administrators to restrict users' access and limit
 the operations they can perform. For instance, you could give your database operator access only
 to the database-related actions.
```

Comment:
Same as in LDAP PR, I think it worth referring to https://github.com/stackstorm/st2-rbac-backend repository URL somewhere at start of the RBAC doc section so users will be able to find it and look at the source code if they need.

Reply:
@hnanchahal we'll still need to include the original RBAC repository URL in st2docs, see above.

That's only blocker for this PR.